lib/entry/LiveEntry.js

```diff
@@ -130,6 +131,30 @@ function saveFlavorDownloadersConfiguration() {
     });
 }
 
+function getEntryServerNodeId() {
+    var getLiveEntryServerNodeIdPromise =
```

Comment:
Also this looks like a duplicate of StateManager::getEntryServerNodeId
I suggest to make the suggested change and move it to a shared code location